wallet/wallet.go

```diff
@@ -95,3 +124,24 @@ func (m *Module) SendFil(ctx context.Context, from string, to string, amount *bi
 	_, err = m.api.MpoolPushMessage(ctx, msg)
 	return err
 }
+
+// FundFromFaucet make a faucet call to fund the provided wallet address.
+func (m *Module) FundFromFaucet(ctx context.Context, addr string) error {
```

Comment:
If makes sense in the future, we can make a pluggable component that might fund wallets by an external service, so this can be extendable.